client/components/paypalPayments/oneTimePayment/typescript/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "paypal-vanilla-typescript-example",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "scripts": {
+    "start": "vite",
+    "build": "tsc --build && vite build",
+    "preview": "vite preview",
+    "format": "prettier . --write",
+    "format:check": "prettier . --check",
+    "typecheck": "tsc --build --noEmit",
+    "lint": "echo \"eslint is not set up\""
+  },
+  "author": "",
+  "license": "MIT",
+  "description": "",
+  "dependencies": {
+    "@paypal/paypal-js": "^8.3.0"
```

Comment:
@EvanReinstein when you get a chance, can you update the PR description to remove the mention of `npm link` since we no longer need it?